WordPress/Classes/ViewRelated/Stats/JetpackSettingsViewController.m

```diff
@@ -130,13 +146,6 @@ - (void)viewDidLoad
     [self addSkipButtonIfNeeded];
 }
 
-// This resolves a crash due to JetpackSettingsViewController previously using a .xib.
-// Source: http://stackoverflow.com/questions/17708292/not-key-value-coding-compliant-error-from-deleted-xib
-- (void)loadView
-{
-    [super loadView];
-}
-
```

Comment:
I think the only reason it was here was for crashes happening with the simulator with the old XIB hanging around. The crash never would have happened with an App Store deploy or if the simulator would have just been reset.